pkg/internal/hexutils/hexutils.go

```diff
@@ -0,0 +1,35 @@
+package hexutils
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+// Decode decodes a hex string with 0x prefix.
+func Decode(input string) ([]byte, error) {
```

Comment:
Non-blocking of course but just noting we could move this code to `keep-common` at some point and use it [here](https://github.com/keep-network/keep-common/pull/116#discussion_r1181538174).

Reply:
Side note: we can use strings.TrimPrefix to do prefix removal.

Reply:
We would need to trim `0x` and `0X`, so I'm not sure which approach would be better.